src/components/DiffView/index.spec.tsx

```diff
@@ -627,12 +656,105 @@ describe(__filename, () => {
     );
   });
 
-  it('does not render widgets without linter messages', () => {
-    const widgets = renderAndGetWidgets({
-      diff: parseDiff(diffWithDeletions)[0],
-      selectedMessageMap: null,
+  describe('comment list', () => {
+    it('renders inline comments for normal and inserted lines', () => {
+      const CommentListResult = () => <div />;
+      const firstLine = nextUniqueId();
+      const secondLine = nextUniqueId();
+      const version = createInternalVersion({
+        ...fakeVersion,
+        id: nextUniqueId(),
+      });
+      const diff = createDiffWithHunks([
+        createHunkWithChanges([
+          {
+            type: 'normal',
+            new_line_number: firstLine,
+            old_line_number: firstLine,
+          },
+          { type: 'insert', new_line_number: secondLine },
+        ]),
+      ]);
+      const widgets = renderAndGetWidgets({
+        diff,
+        version,
+      });
+
+      for (const line of [firstLine, secondLine]) {
+        const widget = renderWidget(diff.hunks, widgets, line);
+
+        const { shell, renderContent } = simulateCommentList({
+          commentList: <CommentListResult />,
+          root: widget,
+        });
+        const commentWidget = renderContent(shell.at(0));
```

Comment:
Since there will only be one `<CommentList>` in the widget, you can simulate it with the default params:

```diff
diff --git a/src/components/DiffView/index.spec.tsx b/src/components/DiffView/index.spec.tsx
index 5ef8bf3..d5b93a6 100644
--- a/src/components/DiffView/index.spec.tsx
+++ b/src/components/DiffView/index.spec.tsx
@@ -683,11 +683,11 @@ describe(__filename, () => {
       for (const line of [firstLine, secondLine]) {
         const widget = renderWidget(diff.hunks, widgets, line);
 
-        const { shell, renderContent } = simulateCommentList({
+        const { renderContent } = simulateCommentList({
           commentList: <CommentListResult />,
           root: widget,
         });
-        const commentWidget = renderContent(shell.at(0));
+        const commentWidget = renderContent();
         expect(commentWidget.find(CommentListResult)).toHaveLength(1);
 
         const commentList = widget.find(CommentList);
```